src/components/common/ModalTutorial.vue

```diff
@@ -51,7 +51,7 @@
                   ? `Read the full guide`
                   : `Next step`
               }}
-              <i class="material-icons arrow_forward"></i>
+              <i class="material-icons arrow_forward" :class="currentStep === steps.length ? `final-step` : ``"></i>
```

Comment:
i would make `currentStep === steps.length` a computed property so this will not be duplicated

Reply:
Good point!

Reply:
This is done @jbibla ;-)

Reply:
> noticed some small opportunities for improvement:
> 
> * [x]  clicking next or back resets the scroll position on the page
> * [ ]  last button should use the upward diagonal arrow to imply you are leaving the app
> * [ ]  mobile button placement is a bit funny and should be fixed
> 
> future:
> 
> * how do we scale these to every network from a technical perspective
> * include links to exchanges, ledger store, iOS, Android, extension

Let's merge it so we can refactor in another PR to allow including links in the content using dynamic named slots.

To scale tutorials for other networks we can:

* Refactor ModalTutorial component to use slots (so we have links!)
* Create individual tutorial components that import ModalTutorial component and have it  content inside. Easy to maintain and allow to open the same tutorial from a different component without duplicating code.
* Cause of the differences between networks, my proposal is to create a directory hierarchy for tutorials like:

```
src/components/tutorials/cosmos/governance.vue
src/components/tutorials/cosmos/staking.vue
src/components/tutorials/cosmos/keys.vue
src/components/tutorials/cosmos/tokens.vue
...
src/components/tutorials/polkadot/governance.vue
src/components/tutorials/polkadot/staking.vue
src/components/tutorials/polkadot/keys.vue
src/components/tutorials/polkadot/tokens.vue
...
```